app/views/organizations/adoptable_pets/show.html.erb

```diff
@@ -60,7 +60,49 @@
           </div>
         <% end %>
 
-        <%= render partial: "shared/pet_card_bigger", pet: @pet %>
+        <%= render CardComponent.new(image_options: {default: ""}) do |c| %>
+          <%= c.with_header do %>
+            <div class="d-flex justify-content-between align-items-center">
+              <div>
+                <h4 class="mb-0"><%= "#{t('general.about')} #{@pet.name}" %></h4>
+              </div>
+
+              <% if current_user&.adopter_foster_account %>
+                <div id="like_button_pet_<%= @pet.id %>" class="d-flex flex-column align-items-center">
+                  <%= render LikeButtonComponent.new(pet: @pet, like: @likes&.find { |lp| lp.pet_id == @pet.id }) %>
+                </div>
+              <% end %>
+            </div>
+          <% end %>
+          <%= c.with_body do %>
+            <p><%= @pet.description %></p>
+
+            <ul class="list-group list-group-flush">
+              <% pet_infos = {"Age": time_ago_in_words(@pet.birth_date).titleize,
+                              "Sex": @pet.sex,
+                              "Breed": @pet.breed,
+                              "Weight": "#{@pet.weight_from} to #{@pet.weight_to} #{@pet.weight_unit}"
+              } %>
+              <% pet_infos.each do |pet_info| %>
+              <li class="list-group-item px-0">
+                <div class="d-flex justify-content-between align-items-center">
+                  <div class="d-flex align-items-center">
+                    <i class="fe fe-book me-2"></i>
+                    <div class="ms-2">
+                      <h5 class="mb-0 text-body"><%= pet_info[0] %></h5>
+                    </div>
+                  </div>
+                  <div>
+                    <div>
+                      <p class="text-dark mb-0 fw-semibold"><%= pet_info[1] %></p>
+                    </div>
+                  </div>
+                </div>
+              </li>
```

Comment:
this loop is a pretty good candiate for pulling out into a component as well.  Can have a renders many, and boil down to something like:
```erb
<% render PetInfos.new do |pi| %>
  <%= pi.item "Age", time_ago_in_words(@pet.birth_date).titleize %>
  <%= pi.item "Sex", @pet.sex %>
  <%= pi.item "Breed", @pet.breed %>
  <%= pi.item "Weight", "#{@pet.weight_from} to #{@pet.weight_to} #{@pet.weight_unit}"
<% end %>
```

Reply:
I like that idea, but I think I will save that exercise for a future PR as this one is mostly refactoring existing partials. I will make a draft for this idea on the projects tab.